docs/changelog.txt

```diff
@@ -1,7 +1,41 @@
 [[changelog]]
 [appendix]
-== Release Notes
+Release Notes
```

Comment:
@mbrukman I submitted #218 to fix this. I was going to do it as CTR so I could include it here but it doesn't look like repo permissions currently allow this (e.g. can't push directly to protected branch, can't merge PR without review and can't self-approve PR).